Makefile

```diff
@@ -104,8 +104,10 @@ install: | prepare
 	$(call message,$(PROJECT_NAME): Installing Drupal)
 	sleep 5
 	$(call docker-www-data, php drush -r /var/www/html/web site-install falcon \
-		--db-url=mysql://$(DB_USER):$(DB_PASSWORD)@$(DB_HOST)/$(DB_NAME) --site-name=$(PROJECT_NAME) --account-pass=admin \
+		--db-url=mysql://$(DB_USER):$(DB_PASSWORD)@$(DB_HOST)/$(DB_NAME) --site-name=$(PROJECT_NAME) \
+		--account-name=$(shell date | md5sum | head -c12) --account-pass=$(shell date +%s | md5sum | head -c12) \
 		install_configure_form.enable_update_status_module=NULL --yes)
+	$(MAKE) -s drush user-block 1
```

Comment:
Nobody will be installing Drupal from this Makefile, why this was done on the Makefile level? We need to block user 1 when the Drupal installation from this profile happens.

Reply:
Disagree - that is not our decision what Drupal (or our installation profile) consumers want to do with user 1. Making decision on what username it should have / what password / disabled or enabled is definitely out of any Drupal distro responsibilities...

Reply:
@duozersk This is a fair point. But that just means that there is no point in development of this story, because there is no point to secure Falcon dev environment on local machines, and it will not get any further than that.